explorer/config-loader.js

```diff
@@ -0,0 +1,23 @@
+var fs = require('fs');
+
+var config = {};
+
+try {
+    var network = process.env.NETWORK || 'testnet';
+    var configFilename = __dirname + '/config.' + network + '.json';
+    var configContents = fs.readFileSync(configFilename);
+    config = JSON.parse(configContents);
+    console.log(configFilename + ' found.');
```

Comment:
I'd remove this log too.

Reply:
this was already there, kind of usefull to know with file was picked up